lib/trino-cache/src/test/java/io/trino/cache/TestEvictableCache.java

```diff
@@ -585,4 +588,26 @@ public void testPutOnNonEmptyCacheImplementation()
                 .isInstanceOf(UnsupportedOperationException.class)
                 .hasMessage("The operation is not supported, as in inherently races with cache invalidation");
     }
+
+    @RepeatedTest(1000)
```

Comment:
omg

Reply:
Just concurrency issues things, I think it worth keeping like that as is was easier to reproduce locally with it, and test itself takes 1-2ms to execute